docker/start-agent.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+export PBM_MONGODB_URI="mongodb://${PBM_AGENT_MONGODB_USERNAME}:${PBM_AGENT_MONGODB_PASSWORD}@localhost:${PBM_MONGODB_PORT}/?replicaSet=${PBM_MONGODB_REPLSET}"
+
+OUT="$(mktemp)"
+timeout=5
+
+for i in {1..10}; do
```

Comment:
are 50 seconds enough to build a replicaset?
can we pass `initialDelaySeconds` https://github.com/percona/percona-server-mongodb-operator/blob/master/pkg/apis/psmdb/v1/psmdb_defaults.go#L150 via env var and use it as default timeout?

Reply:
I multiply timeout on the current iteration number. So, in fact, the wait time might be up to ~4,5 min.